learn/statements.md

```diff
@@ -0,0 +1,115 @@
+
+2. Conditional Statement
+
+```python
+x = 3
+if x < 5:
+    print("x is less than 5")
+else:
+    print("x is greater than or equal to 5")
+```
+
+In this example, the if-else statement is used to check the value of x and `print` a corresponding message. 
+
+By using statements, programmers can instruct the computer to perform a variety of tasks, from simple arithmetic operations to complex decision-making processes. Proper use of statements is crucial to writing efficient and effective Python code.  
+
+## Statement Set  
+
+Here's a table summarizing various types of statements in Python:
+
+| Statement                | Description                                                       |
+|--------------------------|-------------------------------------------------------------------|
+| Multi-Line Statements    | Statements spanning multiple lines using line continuation or braces.|
+| Compound Statements      | Statements that contain other statements (e.g., if, while, for).   |
+| Simple Statements        | Basic standalone statements that perform a single action.          |
+| Expression Statements    | Statements that evaluate and produce a value.                      |
+| pass Statement           | A placeholder statement that does nothing.                        |
+| del Statement            | Used to delete references to objects.                              |
+| return Statement         | Terminates a function and returns a value (optional).              |
+| import Statement         | Imports modules or specific objects from modules.                  |
+| continue and break Statements | Control flow statements used in loops (continue skips to the next iteration, break exits the loop). |
+
+Please note that this table provides a brief overview of each statement type, and there may be additional details and variations for each statement.
+
+## Multi-Line Statements  
+
+Multi-line statements are a convenient way to write long code in Python without making it cluttered. They allow you to write several lines of code as a single statement, making it easier for developers to read and understand the code. Here are two examples of multi-line statements in Python:
+
+1. Using backslash:
+
+```python
+total = 10 + \
+        20 + \
+        30
+print(total) ### Output
+
+```
+
+2. Using parentheses:
+
+```python
+fruit_list = ('Apple',
+              'Mango',
+              'Banana',
+              'Orange')
+print(fruit_list) ### Output
+
+```
+
+## Simple Statements  
+
+Simple statements are the smallest unit of execution in Python programming language and they do not contain any logical or conditional expressions. They are usually composed of a single line of code and can perform basic operations such as assigning values to variables, printing out values, or calling functions.
+
+Examples of simple statements in Python:
+
+```python
+### Assigning a value to a variable
+
```

Comment:
1. Why we use `###` instead of a single `#` in a code comments?
2. Remove an empty line between a comment and a following code line.